tests/test_node_mono_repo.py

```diff
@@ -139,6 +139,32 @@ def test_generate_index_ts():
         assert filecmp.cmp(generated_index_path, sample_index_path)
 
 
+def test_write_release_please_config():
+    # use a non-nodejs template directory
+    with util.chdir(FIXTURES / "node_templates" / "release_please"):
```

Comment:
I would use `util.copied_fixtures_dir` instead. It copies the fixtures to a temporary directory and runs in that temporary directory context. This way, it won't modify the fixture in place.